src/mudlet.pro

```diff
@@ -34,8 +34,8 @@
 #                                                                          #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5)|if(lessThan(QT_MAJOR_VERSION,6):lessThan(QT_MINOR_VERSION, 6)) {
-    error("Mudlet requires Qt 5.6 or later")
+lessThan(QT_MAJOR_VERSION, 5)|if(lessThan(QT_MAJOR_VERSION,7):lessThan(QT_MINOR_VERSION, 7)) {
```

Comment:
:stop_sign: This is wrong! Only the **last** of the three conditions needs changing from `6` to `7` - the first causes rejection of any Qt 4.x version, the middle one ***then*** enables the **last** one if the major number is `5` and *that* requirement has not changed.

Reply:
Updated.